test/tests_for_sdkmr/ee2_retry_test.py

```diff
@@ -60,12 +60,12 @@ def test_retry_db_failures():
     # One DB failure
     rj._db_update_failure = MagicMock(side_effect=Exception("Boom!"))
     with raises(Exception):
-        rj._retry(job_id=retry_job.id, job=retry_job, parent_job=parent_job)
+        rj._retry(job_id=retry_job.id, job=retry_job, batch_job=parent_job)
```

Comment:
We could rename the parent_job here to retry parent but it's maybe not in scope for this pr

Reply:
Ugh, I just missed it. There's a lot of `parent_job*` stuff scattered around the tests

Reply:
Here i think it meant retry_parent